[
    {
        "type": "message",
        "user": "U1JB7QCBD",
        "text": "what is the relationship between ibm-blockchain-js and the fabric node sdk?",
        "ts": "1467633373.000973"
    },
    {
        "type": "message",
        "user": "U0KM61BCP",
        "text": "the ibm one inspired the hyperledger version",
        "ts": "1467636234.000974"
    },
    {
        "type": "message",
        "user": "U0KM61BCP",
        "text": "the ibm one is slightly dated now",
        "ts": "1467636248.000975"
    },
    {
        "type": "message",
        "user": "U1JB7QCBD",
        "text": "I see, because i was trying out the marbles demo, and i was not sure how much of what I saw there applies to the node sdk",
        "ts": "1467636270.000976"
    },
    {
        "type": "message",
        "user": "U1JB7QCBD",
        "text": "but given what you said, I think I should just start from scratch",
        "ts": "1467636300.000977"
    },
    {
        "type": "message",
        "user": "U0P75RFT4",
        "text": "<@U1JK24HE3>: try setting CORE_VM_ENDPOINT=unix:\/\/\/var\/run\/docker.sock",
        "ts": "1467639690.000978"
    },
    {
        "type": "message",
        "user": "U1JK24HE3",
        "text": "<@U0P75RFT4>: thank you, it worked! The peer has communication with the docker daemon now",
        "edited": {
            "user": "U1JK24HE3",
            "ts": "1467649356.000000"
        },
        "ts": "1467648931.000979"
    },
    {
        "type": "message",
        "user": "U1JK24HE3",
        "text": "Instead of the \"Error starting container: cannot connect to Docker endpoint\u201d, I now get the following error:\n\n```\n{\"transactionResults\": [\n            {\n                \"error\": \"Error starting container: Tag latest not found in repository <http:\/\/docker.io\/hyperledger\/fabric-baseimage|docker.io\/hyperledger\/fabric-baseimage>\",\n                \"errorCode\": 1,\n                \"uuid\": \"9333f022-e2f5-42da-b42c-088e420749d5\"\n            }\n        ]\n}\n```\n\nTo try and fix this i built the base image and tagged it as latest\n\n```\n$ cd $FABRIC\/images\/base &amp;&amp; make docker\n&gt; ...\n&gt; Successfully built 7d600d503e09\n\n$ docker tag 7d600d503e09 hyperledger\/fabric-baseimage:latest\n$ docker images\nREPOSITORY                      TAG                 IMAGE ID            CREATED             SIZE\nhyperledger\/fabric-baseimage    latest              7d600d503e09        19 minutes ago      1.061 GB\nhyperledger\/fabric-baseimage    x86_64-0.0.10       7d600d503e09        19 minutes ago      1.061 GB\n```\n\nIt doesnt give me that error anymore, Now i get this error...\n\n```\n{\"transactionResults\": [\n            {\n                \"error\": \"Error starting container: cannot normalise nothing\",\n                \"errorCode\": 1,\n                \"uuid\": \"a2253f1e-ffbe-4bb1-8f8c-d4c001cef572\"\n            }\n        ]\n}\n```\n\nAny ideas on what could be going on? Did tagging it myself actually _fix_ anything or is there something else going on?\n\n*update*: looks like the \u201ccannot normalise nothing\u201d error might be a docker for mac bug. googling \u201ccannot normalise nothing\u201d only comes up with one result:\n\n<https:\/\/forums.docker.com\/t\/problems-with-new-beta-release-1-12-0-rc2-beta16-build-9493\/16731>",
        "edited": {
            "user": "U1JK24HE3",
            "ts": "1467703722.000000"
        },
        "ts": "1467652763.000981"
    },
    {
        "user": "U1N4KNWP2",
        "type": "message",
        "subtype": "channel_join",
        "text": "<@U1N4KNWP2|thomas.leplus> has joined the channel",
        "ts": "1467679109.000982"
    },
    {
        "type": "message",
        "user": "U11QC59MY",
        "text": "Is it possible for a given VP to have the data within its premises(only to that VP), but not to other VPs? I am visioning like each VP has some data(not everything) to itself, not sharing with  other VPs\/network.",
        "ts": "1467692856.000983"
    },
    {
        "type": "message",
        "user": "U1751P7U4",
        "text": "is it possible to get the block number which contains one particular transaction?or do i have to scan all the blocks to find that particular transaction",
        "ts": "1467700382.000984"
    }
]